lambdatimes/src/components/Header.js

```diff
@@ -1,16 +1,58 @@
 import React from 'react';
-
-// Refactor this component to use styled components and not classNames. 
+import styled from 'styled-components'
+// Refactor this component to use styled components and not classNames.
 // You can find the corresponding CSS in the CSS/index.css file
 
+const HeaderWrapper = styled.div`
+display: flex;
+justify-content: space-between;
+align-items: center;
+flex-direction: row;
+background-color: #fff;
+border-bottom: 1px solid lightgrey;
+width: 100%;
+height: 120px;
+margin-top: 44px;
+padding-bottom: 15px;
+`
+
+const HeaderDate = styled.span`
```

Comment:
Good work with Styled Components. You've repeated some code here with the Spans. I wonder if there is a way to not repeat as much?